networkx/algorithms/centrality/dispersion.py

```diff
@@ -69,12 +69,12 @@ def _dispersion(G_u, u, v):
                 norm_disp = ((total + b) ** alpha) / (embededness + c)
             else:
                 norm_disp = (total + b) ** alpha
-            dispersion = norm_disp
+            disper = norm_disp
```

Comment:
Why are we making this abbreviation? Is there a name collision problem here?
Seems better to write out the name, especially if the function name has an underscore in front.

Reply:
The method `_dispersion` is inside `dispersion`, so we should probably avoid naming a variable the same as the outer function name.

Reply:
How about:
```
        if normalized:
            if embededness + c != 0:
                disp = ((total + b) ** alpha) / (embededness + c)
            else:
                disp = (total + b) ** alpha

        else:
            disp = total

        return disp
```